app/src/main/res/layout/activity_login.xml

```diff
@@ -102,13 +102,91 @@
             android:ems="10"
             android:hint="@string/password"
             android:inputType="textPassword"
-            android:visibility="invisible"
+            android:visibility="gone"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/username"
             app:layout_constraintWidth_max="280dp"
             tools:text="Password" />
 
+        <Button
+            android:id="@+id/showAdvanced"
```

Comment:
The loading spinner when clicking do login is behind the show advanced button

Reply:
The entire advanced settings block now shows / hides along with the login button, to prevent it from overlapping with the spinners.